src/main/java/org/spongepowered/api/service/permission/PermissionService.java

```diff
@@ -64,6 +64,11 @@
      * subject is at the root of all inheritance trees, above even subject type-specific
      * defaults, meaning it has the lowest priority when all other weighting is equal.
      *
+     * <p>Note: This data should be persisted, so plugins that add permissions to this subject
+     * must take care to not override permissions already set or modified. It is also
+     * recommended to use {@link Subject#getTransientSubjectData()} where possible to
+     * avoid persisting unnecessary data.
```

Comment:
I agree with you - however, the current intended behaviour is that defaults are persisted. 

(at least, thats what I'm assuming, judging from the responses I've had from blood, and from looking at the PEX implementation. I'm all for saying "nope, defaults should be transient", however this PR was about improving documentation, not changing intended behaviours.) 

There's missing `</p>` tags throughout PermissionService. You'll notice they don't appear in the existing JavaDocs either, so I just followed the same convention.